app/controllers/api/v2/search_controller.rb

```diff
@@ -14,7 +14,9 @@ def suggestions
       private
 
       def matching_suggestions
-        return SearchSuggestion.fuzzy_search(params[:q]) if params[:q].present?
+        if !SearchService::RogueSearchService.call(params[:q]) && params[:q].present?
```

Comment:
Should check whether `params[:q]` exists before using it
```suggestion
        if params[:q].present? && !SearchService::RogueSearchService.call(params[:q])
```